exec/vector/src/main/java/org/apache/drill/exec/exception/OversizedAllocationException.java

```diff
@@ -27,6 +27,7 @@
  * {@link RecordBatch#next() iteration}.</p>
  *
  */
+@SuppressWarnings("serial")
```

Comment:
Will it be better to provide `serialVersionUUID` instead of warning suppression?